jax-controlnet-sprint/README.md

```diff
@@ -74,3 +75,102 @@ We will have talks from folks working at JAX & TPU teams at Google, diffusers an
 |Margaret Mitchell, Hugging Face	| Ethics of Text-to-Image |	5.20pm-6.00pm CEST / 08.20am-09.00am PST	| [![Youtube](https://www.youtube.com/s/desktop/f506bd45/img/favicon_32.png)](https://www.youtube.com/watch?v=SOj2sxgvFe0) |
 
 [signup form]: https://forms.gle/t3M7aNPuLL9V1sfa9
+
+## Data and Pre-Processing
+
+In this section, we will cover how to build your own dataset for ControlNet training.
+
+### Prepare a large local dataset
+
+#### Mount a disk 
+
+If you need extra space, you can follow [this guide](https://cloud.google.com/tpu/docs/setup-persistent-disk#prerequisites) to create a persistent disk, attach it to your TPU VM, and create a directory to mount the disk. You can then use this directory to store your dataset.
+
+#### Data preprocessing 
+
+Here we demonstrate how to prepare a large dataset to train a ControlNet model with canny edge detection. More specifically, we provide an [example script](./coyo_1m_dataset_preprocess.py) that:
+* Select 1 million image-text pairs from an existing dataset [COYO-700M](https://huggingface.co/datasets/kakaobrain/coyo-700m)
+* Download each image and use Canny edge detector to generate the conditioning image. 
```

Comment:
What conditioning image will be generated? Pose, Canny? Give some hint about that here.

Reply:
```suggestion
* Downloads each image and use Canny edge detector to generate the conditioning image. 
```